app/sessionStore.js

```diff
@@ -233,6 +233,8 @@ module.exports.cleanAppData = (data, isShutdown) => {
   data.temporarySiteSettings = {}
   // Delete Flash state since this is checked on startup
   delete data.flashInitialized
+  // Delete Recovery status on shut down
+  delete data.ui.about.preferences.recoverySucceeded
```

Comment:
Yeah, should I just wrap it in an if check or do other operations need this too?